plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -148,7 +148,7 @@ func (r *NodeAuthorizer) Authorize(ctx context.Context, attrs authorizer.Attribu
 // authorizeStatusUpdate authorizes get/update/patch requests to status subresources of the specified type if they are related to the specified node
 func (r *NodeAuthorizer) authorizeStatusUpdate(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
 	switch attrs.GetVerb() {
-	case "update", "patch":
+	case "get", "update", "patch":
```

Comment:
is get of /status actually used by the kubelet? if not, drop it from the description rather than add permissions we don't use

Reply:
after analyzing the audit log, I found there is no get used by kubelet.

Reply:
thanks for investigating. this authorizer is limited to requests actually made by the kubelet

/close